lib/chef/provider/mount/windows.rb

```diff
@@ -41,8 +41,10 @@ def initialize(new_resource, run_context)
         def load_current_resource
           if is_volume(@new_resource.device)
             @mount = Chef::Util::Windows::Volume.new(@new_resource.name)
```

Comment:
Is there a reason we don't use new_resource.mount_point here? It's the name property so it seems like we're working around the core bug without actually fixing it